pandas/tests/frame/test_query_eval.py

```diff
@@ -1091,7 +1104,56 @@ def test_mixed_underscores_and_spaces(self, df):
         expect = df["A"] + df["D_D D"]
         tm.assert_series_equal(res, expect)
 
-    def backtick_quote_name_with_no_spaces(self, df):
+    def test_backtick_quote_name_with_no_spaces(self, df):
         res = df.eval("A + `C_C`")
         expect = df["A"] + df["C_C"]
         tm.assert_series_equal(res, expect)
+
```

Comment:
can you create a new test class (and move all of the quoting testst here)

Reply:
All my tests regarding quoting are already in one class: TestDataFrameQueryBacktickQuoting

Reply:
k great